content/uc-doc/upgrade/upgrade_notes.md

```diff
@@ -2,6 +2,18 @@
 title: Upgrade notes
 ---
 
+## 20.10 {#20.10}
+
+- The TLS configuration has been deprecated on the following services. You should always use NGINX
+  to proxy communication with wazo-platform services. To follow this change, the listen address has
+  been changed to 127.0.0.1 by default.
+
+  - wazo-ui
+
```

Comment:
IMO we should not add this upgrade note (this missing was intended)
* probably 99.9% of people using wazo-ui was already behind reverse proxy
* I never saw any publishing (blog, forum, chat) that say to use wazo-ui directly through port
* it make noise (still IMO)

Reply:
Agreed.